sdk/appconfiguration/azure-data-appconfiguration/src/test/resources/session-records/aadAuthenticationAzConfigClient.json

```diff
@@ -4,7 +4,7 @@
     "Uri" : "https://appconfigtestshawnfang.azconfig.io/kv/newKey?api-version=1.0",
```

Comment:
Remove `*.json` files where test cases have not changed. This adds unnecessary diffs to the PR.